notebooks/index.ipynb

```diff
@@ -65,7 +65,7 @@
     "\n",
```

Comment:
<p>Is there a link missing on the "available here" part?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>it was supposed to mean here as in here on this website you are looking at right now. should I rephrase it?</p>

Reply:
<p>maybe write: "The notebooks for each class will be made available from links on this website."?</p>